pretty_tables/tables.py

```diff
@@ -1,53 +1,119 @@
-class PrettyTables():
-    @classmethod
-    def generate_table(cls, headers, rows, empty_cell_placeholder=None):
-        """Generate pretty tables with headers and rows (arrays).
+"""The tables module for the pretty_tables library."""
+
+
+class TableColors:
+    """A class to contain constants for colors for the table."""
+
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class PrettyTables(object):
+    """A class used to generate pretty tables for console output."""
+
+    @staticmethod
+    def generate_table(headers, rows, empty_cell_placeholder=None, colors=None, truthy=None):
+        """
+        Generate pretty tables with headers and rows (arrays).
+
         We do this by validating the headers and rows are valid arrays
         and are set, then convert all items to strings and append
         them to their respective row, then append each row to the table.
         We replace empty items with a placeholder and finally return
         the validated, generated, formatted table.
 
-        headers = 1d array
-        rows = 2d array
-        empty_cell_placeholder = whatever you want in place of an empty cell
+        Args:
+            headers (list): a list of headers for the table.
+            rows (list[list]): A list of rows for the table.
+            empty_cell_placeholder: The value to be placed in cells with no data.
+            colors (list): A list of TableColors for each row.
+            truthy (int): Index of the column to be compared for truthy values. If truthy is not None
+                truth comparison will override all other formats.
         """
-        cls._validate_table_input(headers, rows)
-        table = []
-        table.append(headers)
+        PrettyTables._validate_table_input(headers, rows)
+        table = [headers]
         for row in rows:
             formatted_row = []
             for item in row:
                 if item is True:
-                    data = 'True'
+                    data = True
```

Comment:
Can you explain what you mean?

Reply:
For instance, if the user passes a boolean `True` to the table, I want to actually print the string `True` and not just have that field be truthy. Same for `False`, the string `False` should appear.

Reply:
You'll want to rebase against `main` here, I was silly and created this dumb shim initially when really all I needed was to coerce them to a string in the end instead of doing this. This chunk of code has been refactored since.

Reply:
I rebased. I hate rebasing

Reply:
I don't believe this was rebased properly as the `true/false` logic shouldn't be here anymore. I realized (duh) that I should have just made the boolean a string when printing and I wouldn't need this piece. Let's remove it (unless I'm overlooking a reason to add it back, I assume it's still here in this diff because of a rebase gone wrong) so that we don't reintroduce previous behavior accidentally.